app/services/config_report_importer.rb

```diff
@@ -0,0 +1,41 @@
+class ConfigReportImporter < ReportImporter
+  attr_reader :report
+  def self.authorized_smart_proxy_features
+    ['Puppet']
+  end
+
+  def report_name_class
+    ConfigReport
+  end
+
+  def import
+    @report = super
```

Comment:
I thought it was shared.  If you use `report` (the attr_reader) rather than `@report`, is it better?